lib/api/apiUtils/bucket/getNotificationConfiguration.js

```diff
@@ -0,0 +1,33 @@
+const { errors, models } = require('arsenal');
+const { NotificationConfiguration } = models;
+
+const { config } = require('../../../Config');
+
+function getNotificationConfiguration(parsedXml) {
+    const targets = [];
+    if (!config.bucketNotificationDestinations) {
+        return { error: errors.InvalidArgument.customizeDescription(
+            'Unable to validate the following destination configurations') };
+    }
+    config.bucketNotificationDestinations.forEach(t => {
+        targets.push(t.resource);
+    });
+    const notifConfig = new NotificationConfiguration(parsedXml).getValidatedNotificationConfiguration();
+    if (notifConfig.error) {
+        return notifConfig;
+    }
+    const notifConfigTargets = [];
+    notifConfig.queueConfig.forEach(t => {
+        const arnArray = t.queueArn.split(':');
+        notifConfigTargets.push(arnArray[5]);
+    });
```

Comment:
for each and push is just a map.
```suggestion
    const notifConfigTargets = notifConfig.queueConfig.map(t => t.queueArn.split(':')[5]);
```